app/views/oregon_digital/base/_local_contexts.html.erb

```diff
@@ -8,7 +8,9 @@
     <%# FETCH: Get the array of the local contexts from work/collection %>
     <% if location_name.eql?('collection') %>
       <h2 class='h6'>Local Contexts</h2>
-      <%# DISPLAY COLLECTION CODE WILL GO HERE ON ANOTHER PR %>
+      <% @local_contexts_labels = OregonDigital::LocalContextsService.new.all_labels(collection_lc.local_contexts) %>
+      <% @lc_labels = @local_contexts_labels %>
```

Comment:
Extra assignment?

Reply:
I have to pass in an extra `assignment` as the row display at the bottom of the code utilize the `@lc_labels`. So to make sure that one section on display work for both `work & collection`, I have to do that extra assignment to make the display universal usage in both area of `work & collection`.